### ISSUE TITLE
NTR: 3,4-dihydroxybutyric acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8611]
New term request 3,4-dihydroxybutyric acid high in urine (segal_030513181350):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000608
- label: Elevated urinary 3,4-Dihydroxybutyric acid level
- parents: [Abnormal circulating monocarboxylic acid concentration (HP:0010996)](https://hpo.jax.org/app/browse/term/HP:0010996)
- pmid: PMID:12127325
- def: The amount of 3,4-Dihydroxybutyric acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8611
- synonyms:
